ShipGame/.vscode/launch.json

```diff
@@ -0,0 +1,32 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "name": "ShipGame Android Debug",
+            "type": "dotnet",
```

Comment:
This wont work out of the box.

Reply:
I just built using the dotnet tools and deployed the built apk to Android, worked fine